gradle.properties

```diff
@@ -1,5 +1,5 @@
 version=unspecified
-jnaVersion=5.5.0
+jnaVersion=5.10.0
 javaFxPlatform=unspecified
 javaFxVersion=11.0.1
-log4j2Version=2.13.3
+log4j2Version=2.16.0
```

Comment:
Since the variable isn't used anymore, this can be removed, too

Reply:
It's still used by `org.apache.logging.log4j:log4j-api`